### DISCUSSION TITLE
Tracking static objects

### DISCUSSION THREAD

**Discussion by adielstatman** [dstl/Stone-Soup#613] [Q&A]
Hi.
I have static objects, each with an i.d., that I want to track.
Without using their ids, but with the assumption that all of them are static, which transition model should I use?
Currently I'm using ConstantVelocity(0.1) and get flips of ids after several time points, in most of the tracks.
Thanks!

**Comment by sdhiscocks** [comment]
If you know your targets are static, you could change your state space to position only, and use the [`RandomWalk`](https://stonesoup.readthedocs.io/en/v0.1b8/stonesoup.models.transition.html#stonesoup.models.transition.linear.RandomWalk) model with very small value for the noise coefficient.
